### PR TITLE
clickhouse dbt demo

### PR DESCRIPTION
**Pull request by runwuf**
This demostrates how to integrate dbt with clickhouse and run a Simple View Materialization demo.

### CODE REVIEW

**Comment by aris-aiven** [comment]
Hey @runwuf, I haven't forgotten about your PR and will try to look at it and run it tomorrow (otherwise beginning of next week)

**Comment by runwuf** [comment]
No worries, thanks so much @aris-aiven !

File: solutions/clickhouse-dbt/ch-dbt.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+CH_HOST=$(echo ${CH_JSON} | jq -r '.components[] | select(.component=="clickhouse") | .host')
+CH_PORT=$(echo ${CH_JSON} | jq -r '.components[] | select(.component=="clickhouse") | .port')
+CH_HTTPS_PORT=$(echo ${CH_JSON} | jq -r '.components[] | select(.component=="clickhouse_https") | .port')
+
+CH_CLI="./clickhouse client --user ${CH_USER} --password ${CH_PASS} --host ${CH_HOST} --port ${CH_PORT} --secure"
+}
+
+ch-dbt_setup() {
+pip install dbt-core
+pip install dbt-clickhouse
```
Comment (aris-aiven) [inline solutions/clickhouse-dbt/ch-dbt.sh:19]: Would be create to have these packages installed in a virtual env

File: solutions/clickhouse-dbt/ch-dbt.sh
Comment (aris-aiven) [inline solutions/clickhouse-dbt/ch-dbt.sh:1]: Would be neat to reformat the file to ident the functions, would greatly improve readability

**Review by aris-aiven — CHANGES_REQUESTED** [review]
Awesome work @runwuf!

Left two stylistic comments but except for that it's good to merge on my side

**Review by aris-aiven — APPROVED** [review]
Looks great, thank you for the demo!

**Pull request merged**